images/linux/scripts/installers/java-tools.sh

```diff
@@ -24,11 +28,8 @@ if isUbuntu16 || isUbuntu18 ; then
     apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
     apt-add-repository "deb https://repos.azul.com/azure-only/zulu/apt stable main"
     apt-get update
-    apt-get -y install zulu-7-azure-jdk=\*
     # Open JDP Adopt does not exist for Ubuntu 20
-    apt-get -y install adoptopenjdk-12-hotspot=\*
-    echo "JAVA_HOME_7_X64=/usr/lib/jvm/zulu-7-azure-amd64" | tee -a /etc/environment
-    DEFAULT_JDK_VERSION=8
+    DEFAULT_JDK_VERSION=$JAVA_DEFAULT
```

Comment:
I think we can remove from this block this variable. Pls pay attention at the line 37.